### PR TITLE
Add year tabs

### PR DESCRIPTION
**Pull request by rahuljoban**
Added tabs for multiple years of data

### CODE REVIEW

File: employee.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                        {% elif star = 'half' %}
+                            fa-star-half-o half
+                        {% else %}
+                            fa-star-o empty
+                        {% endif %}"></i>
+                    {% endfor %}
+                    </div>
+                    <div class="message">
+                    {% if employee.num_ratings < 5 %}
+                        {{ ratings_needed }} more votes needed
```
Comment (blx) [inline employee.html]: If this were phrased as
```html
{% if ratings_needed > 0 %}
    {{ ratings_needed }} more votes needed
```
then you could remove the magic constant `5`, which would further separate the frontend from the backend and allow that number to be changed in one place in the future, as opposed to two.

**Comment by blx** [comment]
note -- i think this is supposed to be `templates/employee.html`, not `employee.html` at the root level.

If you adjust your branch to reflect that, and then either push a new commit or force push or create a new PR, git will correctly show just your changes here, as opposed to this entirely new file.

File: employee.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+document.getElementById("defaultOpen").click();
+function openSalary(evt, salary) {
+    // Declare all variables
+    var i, tabcontent, tablinks;
+
+    // Get all elements with class="tabcontent" and hide them
+    tabcontent = document.getElementsByClassName("tabcontent");
+    for (i = 0; i < tabcontent.length; i++) {
+
+        tabcontent[i].style.display = "none";
```
Comment (blx) [inline employee.html:199]: jQuery is included on the salary site (it is included in the base template [here](https://github.com/ubyssey/salarydb/blob/master/templates/base.html#L42); you can confirm by opening the console on the page and evaluating `$`), so you can dramatically simplify these loops:

```js
$('.tabcontent').css('display', 'none');

$('.tablinks').removeClass('active');

$('#' + salary).css('display', 'block');

$(evt.currentTarget).addClass('active');
```

File: employee.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+                        <li class="tab1"><a href="javascript:void(0)" class="tab-links" onclick="openSalary(event, 'tab1')"
+                          id="defaultOpen">2016</a>
+                        <li class="tab2"><a href="javascript:void(0)" class="tab-links" onclick="openSalary(event, 'tab2')">2015</a>
+                        <li class="tab3"><a href="javascript:void(0)" class="tab-links" onclick="openSalary(event, 'tab3')">2014</a>
+                    </ul>
+                </div>
+
+                <!-- TAB STYLE -->
+                <style>
+                h3.dept {
```
Comment (blx) [inline employee.html:30]: Ideally this css should go in the static css file (salarydb/static/css/styles.css). At the very least it shouldn't be dangling in the middle of the body 👻

File: employee.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+                  .tab-links li {
+                      float: left;
+                      list-style:none;
+                  }
+
+                  .tab-links a {
+                        text-decoration:  none;
+                        padding:9px 15px;
+                        display:block;
+                        border-radius:3px 3px 3px 3px;
```
Comment (blx) [inline employee.html:80]: this is equivalent to and should be simplified to
```css
border-radius: 3px;
```

**Comment by rahuljoban** [comment]
Thanks for the tips! How do I create a new function with JQuery? When I replaced my for loops with your code, it said that the openSalary method was not defined.

**Pull request closed without merging**